e2e/client/playwright/utils/tree-select-driver.ts

```diff
@@ -1,44 +1,45 @@
 import {Page} from '@playwright/test';
 import {s} from '.';
 
-export const treeSelectDriver = {
-    /**
-     * Get all available options.
-     */
-    async getValues(page: Page): Promise<Array<string>> {
-        await page.locator(s('open-popover')).click();
+export const treeSelectDriver = (dataTestId: string) => {
+    return {
+        /**
+         * Get all available options.
+         */
+        async getValues(page: Page): Promise<Array<string>> {
```

Comment:
> 2. The method is called get getValues and your implementation is for getOptions(which doesn't exist here)

Did you see this one in the [previous comment](https://github.com/superdesk/superdesk-client-core/pull/4519#discussion_r1604661415)?